home/ivp-planning/getting-started/importing-pdfs.md

```diff
@@ -2,30 +2,28 @@
 
 In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.
```

Comment:
This is no longer the case, imported PDFs act as any other object. I think this section can be removed.